docs/servicenow.md

```diff
@@ -0,0 +1,162 @@
+# ticketutil.servicenow
+
+This document contains information on the methods available when working
+with a ServiceNow object. A list of the ServiceNow fields that have been
+tested when creating and editing tickets is included. Because each
+instance of ServiceNow can have custom fields and custom values, some
+of the tested fields may not be applicable to certain instances of ServiceNow.
+Additionally, your ServiceNow instance may contain ticket fields that we have
+not tested. Keep in mind different ServiceNow tables contain different
+fields with different properties. Mandatory fields in one table may be optional
+or non-existent in another table.
+
+Custom field names and values can be passed in as keyword arguments when
+creating and editing tickets. These are processed by the ServiceNow REST API
+right afterwards. You can also use [get_ticket_content](#content) to get ticket
+content including preset field names.
+
+For more information about REST API and fields see ServiceNow's Wiki:<br />
+[REST API](http://wiki.servicenow.com/index.php?title=REST_API)<br />
+[Introduction to Fields](http://wiki.servicenow.com/index.php?title=Introduction_to_Fields)<br />
+[Dot-Walking](http://wiki.servicenow.com/index.php?title=Dot-Walking)
```

Comment:
According to [Markdown syntax specification](https://daringfireball.net/projects/markdown/syntax#p) you can force a line break by adding two or more spaces at the end of the line.

Alternatively, here you could make a list from those three links, or put each of those links on a separate paragraph (requires an empty line between each two links)